docs/source/array-examples.rst

```diff
@@ -0,0 +1,9 @@
+Examples
+========
+
+.. toctree::
+   :maxdepth: 1
+
+   examples/array-random.rst
+   examples/array-numpy.rst
+   examples/array-pickle.rst
```

Comment:
I agree with these objectives.  I think I'm against having them be as prominent in the documents until they're more built out.  What's up here right now feels simplistic.  I would feel more excited about this section if it went over common problems that people actually have, rather than small and artificial examples.  

However, again, content generation like this is probably something that I should handle.  If there are other sections that seem like they're good starts that should be built out then please bring them to my attention.  I'll put this on my todo list for early next week.